src/components/v5/common/ActionSidebar/partials/VerifiedMembersSelect/VerifiedMembersSelect.tsx

```diff
@@ -0,0 +1,184 @@
+import clsx from 'clsx';
+import { utils } from 'ethers';
+import React, { FC } from 'react';
+import { useController, useFormContext, useWatch } from 'react-hook-form';
+
+import { useAdditionalFormOptionsContext } from '~context/AdditionalFormOptionsContext/AdditionalFormOptionsContext';
+import {
+  useRelativePortalElement,
+  useUserByAddress,
+  useToggle,
+  useMobile,
+} from '~hooks';
+import Icon from '~shared/Icon';
+import { formatText } from '~utils/intl';
+import Button from '~v5/shared/Button';
+import SearchSelect from '~v5/shared/SearchSelect/SearchSelect';
+import UserAvatar from '~v5/shared/UserAvatar';
+
+import { AddRemoveMembersType } from '../forms/ManageVerifiedMembersForm/consts';
+
+import { useVerifiedMembersSelect } from './hooks';
+import { VerifiedMembersSelectProps } from './types';
+
+const displayName = 'v5.common.ActionsContent.partials.VerifiedMembersSelect';
+
+const VerifiedMembersSelect: FC<VerifiedMembersSelectProps> = ({
```

Comment:
can we split this one into 2 different ones, one for selecting from already verified members, the other one for selecting from unverified members?
we can reuse styles and everything, just 2 different components which will fetch from different sources
we can mock them until we get the BE up and running :)